Source/WebKit/UIProcess/API/C/WKNavigationResponseRef.h

```diff
@@ -36,6 +36,7 @@ WK_EXPORT WKTypeID WKNavigationResponseGetTypeID();
     
 WK_EXPORT bool WKNavigationResponseCanShowMIMEType(WKNavigationResponseRef);
 WK_EXPORT WKURLResponseRef WKNavigationResponseCopyResponse(WKNavigationResponseRef);
+WK_EXPORT WKFrameInfoRef WKNavigationResponseCopyFrameInfoRef(WKNavigationResponseRef);
```

Comment:
Oh, my bad. Looking at the above `WKNavigationResponseCopyResponse`, it seems that we should drop `Ref` of `WKNavigationResponseCopyFrameInfoRef` .

Reply:
I mean WKNavigationResponseCopyFrameInfoRef → WKNavigationResponseCopyFrameInfo

Reply:
Fixed